server/lib/src/server/migrations.rs

```diff
@@ -928,22 +614,33 @@ impl QueryServerWriteTransaction<'_> {
             // DL4
             SCHEMA_CLASS_OAUTH2_RS_PUBLIC_DL4.clone().into(),
             // DL5
-            SCHEMA_CLASS_PERSON_DL5.clone().into(),
+            // SCHEMA_CLASS_PERSON_DL5.clone().into(),
```

Comment:
why comment them out instead of deleting?

Reply:
Because I'm about to do a second PR to clean it up and wanted to validate some things.